src/models/SqlOutputContentProvider.ts

```diff
@@ -289,36 +289,34 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
 
     // Function to render resultspane content
     public displayResultPane(resultsUri: string, paneTitle: string): void {
-        // Check if the results window already exists
+        // Get the active text editor
         let activeTextEditor = this._vscodeWrapper.activeTextEditor;
-        let previewCommandPromise;
-        let resultPaneColumn;
+
+        // Check if the results window already exists
         if (this.doesResultPaneExist(resultsUri)) {
-            // Implicity Use existing results window by not providing a pane
-            previewCommandPromise = vscode.commands.executeCommand('vscode.previewHtml', resultsUri, paneTitle);
+            // Update the existing results pane with the given reusltsUri
+            this.update(vscode.Uri.parse(resultsUri));
```

Comment:
Aren't we already calling this in another file? Won't this double-reload? I'm basically wondering if we need to do anything in this case